nodes/cma/nodes-cma-autoscaling-custom-trigger-auth.adoc

```diff
@@ -47,20 +61,74 @@ metadata: <1>
   name: secret-cluster-triggerauthentication
 spec:
   secretTargetRef: <2>
-  - parameter: user-name <3>
-    name: secret-name <4>
-    key: USER_NAME <5>
-  - parameter: user-password
+  - parameter: username <3>
+    name: my-secret <4>
```

Comment:
This needs to be `keda-prom-secret` now since it's using the new example secret.